Workbooks/DSTIMWorkbook.json

```diff
@@ -0,0 +1,2286 @@
+{
```

Comment:
It seems DSTIMWorkbook.json format are not as per other workbooks json format. Please refer any other workbooks format  https://github.com/Azure/Azure-Sentinel/tree/master/Workbooks .Thanks